service/src/msr_data_arch.json

```diff
@@ -1209,7 +1304,8 @@
                     "scalar":    1.0,
                     "behavior":  "label",
                     "writeable": true,
-                    "description": "Write 1 to clear the global status bit for PMC0 overflow."
+                    "description": "Write 1 to clear the global status bit for PMC0 overflow. When reading at a higher level domain than its native domain, it aggregates as the count of all such bits that have been set.",
```

Comment:
These types of controls (names like `CLEAR_...` or descriptions like `Write 1 to <do xyz>` typically act like a trigger signal and have no meaningful read-back value.

This reads back as 0 when I try it on our server right after writing 1 to it. The SDM only specifies the write behavior of this MSR, so I think we should not add any specific read expectations to it.

Reply:
I see. Hmmm, that being the case, I'm thinking "expect_same" makes sense for these. That would avoid any confusion from the inconsistent behavior that would result with the "sum" aggregation function and would also get rid of the added "reading" note in the description.

Does this make sense?